src/runtime/Makefile

```diff
@@ -166,6 +166,9 @@ HYPERVISORS := $(HYPERVISOR_ACRN) $(HYPERVISOR_FC) $(HYPERVISOR_QEMU) $(HYPERVIS
 QEMUPATH := $(QEMUBINDIR)/$(QEMUCMD)
 QEMUVALIDHYPERVISORPATHS := [\"$(QEMUPATH)\"]
 
+QEMUSNPPATH := $(QEMUBINDIR)/$(QEMUSNPCMD)
```

Comment:
Sorry, I might have missed something obvious, but where is `QEMUSNPCMD` set?

Reply:
Here https://github.com/kata-containers/kata-containers/pull/6062/files#diff-927b5427ac0d55554f422137273a8aa1361e7a2b197616a2384319085f78bd6b

Reply:
Thanks - that was obvious - I clearer typo'd when searching for it in this PR!